Examples/Windpower Example/windpower-example.py

```diff
@@ -0,0 +1,85 @@
+import json
+import PySAM.Windpower as wp
+
+# create a new windpower model
+wind = wp.new()
+
+# set model inputs from JSON generated by SAM
+with open('windpower-inputs.json', 'r') as file:
```

Comment:
You should use the path library to make sure the input json can be found regardless of the working directory

```
from pathlib import Path

Path(__file__).parent / 'windpower-inputs.json`
```